openapi/paths/subscriptions@{id}@upcoming-invoice.yaml

```
@@ -0,0 +1,31 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Orders
+  summary: Retrieve subscription order's upcoming invoice
+  operationId: GetSubscriptionUpcomingInvoice
+  x-sdk-operation-name: getUpcomingInvoice
+  description: |-
+    Retrieve an upcoming invoice from the specified subscription order.
+
+    The endpoint is temporary before upcoming invoices get a complete
+    integration.
```

nitpick: suggestion based on [writing style guidance for operation descriptions](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)

I also added an updated note about the endpoint status
```suggestion
    Retrieves an upcoming invoice for a specified subscription order.

    > **Note:** This endpoint is temporary until an integration
    is completed for upcoming invoices.
```